src/components/visualizations/chart/chartOptionsBuilder.ts

```diff
@@ -878,12 +912,7 @@ export function generateTooltipTreemapFn(viewByAttribute: any, stackByAttribute:
             textData.push([customEscape(point.category && point.category.name), formattedValue]);
         }
 
-        return `<table class="tt-values gd-viz-tooltip-table">${textData.map(line => (
-            `<tr class="gd-viz-tooltip-table-row">
-                <td class="gd-viz-tooltip-table-cell title gd-viz-tooltip-table-title">${line[0]}</td>
-                <td class="gd-viz-tooltip-table-cell value gd-viz-tooltip-table-value">${line[1]}</td>
-            </tr>`
-        )).join('\n')}</table>`;
+        return renderTooltipHTML(textData);
```

Comment:
Good job with reusing the function instead of a cut & pasted block of HTML code. This is a good improvement to the code readability! 👍

Reply:
thanks :)